f5_openstack_agent/lbaasv2/drivers/bigip/resource_helper.py

```diff
@@ -59,6 +59,7 @@ class ResourceType(Enum):
     ssl_persistence = 33
     universal_persistence = 34
     ssl_cert_file = 35
+    oneconnect = 37
```

Comment:
I think it will become a merge conflict..  A better way is to create branch from your pervious http profile branch, and then append new changes. Anyway, it is your own job to resolve the conflict..

Reply:
@zhang-shengping Conflict has been fixed by I...

Reply:
@zhang-shengping Sorry, I made a wrong comma in the web page.....